### ISSUE TITLE
Game gets stuck on "Loading Part Upgrades" when trying to use the mod

### ISSUE THREAD

**Issue by Arae-1** [Kristallranke/KSPW00tNow#3]
![image](https://github.com/Kristallranke/KSPW00tNow/assets/108728996/d68dfbc4-0451-4863-a60b-73b5b53e37f2)
as shown in the image. I don't know what could be causing this.

**Comment by Githubber616** [comment]
Same here, no loading progress as soon as the plugin is installed, game works fine without.

Wooting (BigBrainAFK | Tony) says this:
the author should use the nuget package of the sdk and not hard copy the code. they should also not ship a dll in their code but rely on the installed analog sdk

**Comment by Kristallranke** [comment]
Sorry for the late response. I looked into this issue some time ago and the fix seemed to be to change the file extension of wooting_analog_wrapper.dll to something different. I wasn't really satisfied with this solution so I didn't commit it and then forgot about it.

Regarding the NuGet package. I copied the code to remove the dependency on Newtonsoft.json like stated in the readme.md. I don't want to bloat the project with dependencies I don't use.
